feugy/tabulous
# feat(web): introduces nice-looking peer pointers

### What's in there?

The initial implementation of peer pointers was barely a draft.
Players deserve something better, like peer's name, and eventually, avatar (why not video?)

Are included here:

- [feat(web): display peer names (and avatar) as their pointer](https://github.com/feugy/tabulous/commit/75bbf03f247cef0412389a5f9ee2ec25daaef82c)
- [fix(web): table and zoom specs are not shared with peers](https://github.com/feugy/tabulous/commit/6bd589f26f1f584a99487001f57aa61af20ab234)
- refactor(web): re-implements peer pointers management as 2D indicators.
- refactor(web): hides indicators which are not in the active camera's frustum

### How to test?

Creates a new game, and invite another player.
Then when the other player joined, see their pointers on your screen. They'll see yours on their.
Don't forget the change the view point, by panning, rotating and zooming your camera. Peer pointers should still be accurately positioned in the absolute 3D coordinates, and not related to your 2D view.

### Notes to reviewers

Please note that indicators **should not have CSS transitions**. This is very sad, but setting transition (event with a duration of 0) introduces a visible delay when panning, rotating or zooming the camera.
